haystack/modeling/model/language_model.py

```diff
@@ -484,6 +484,8 @@ def forward(
         input_ids: torch.Tensor,
         segment_ids: torch.Tensor,
         padding_mask: torch.Tensor,
+        output_hidden_states: bool = False,
+        output_attentions: bool = False,
```

Comment:
Please add docstrings for these new parameters

Reply:
Ok, I have added the docstrings.